docs/platform/concepts/billing-groups.rst

```diff
@@ -1,35 +1,24 @@
 Billing groups
 ==============
 
-Billing groups enable you to set a common billing profile for multiple
-projects and generate a consolidated invoice.
+Billing groups let you to set up common billing profiles in an organization that can then be used across multiple projects. Instead of entering the payment information every time you create a project, you can use the information saved in the billing group. 
 
-Key benefits of billing groups
-------------------------------
+This makes it easier to manage your costs. You receive a consolidated invoice for all projects assigned to a billing group. This means you can, for instance, combine costs based on your organization or IT environment (development, test, production) by creating billing groups for each of these.
 
-The key benefits of using billing groups are:
+You can also track spending by exporting cost information to business intelligence tools using the `invoice API <https://api.aiven.io/doc/#tag/BillingGroup>`_.
 
--  **Receive a single invoice for all your projects.** When you have
-   several Aiven projects that use the same payment method, you can
-   receive a single invoice that consolidates all costs instead of
-   receiving a separate invoice for each project.
+Types of billing groups
+------------------------
 
--  **Group costs based on your organization need.** You can merge costs
-   based on your organization or IT environment (development, test,
-   production) by creating billing groups for related projects. Each
-   billing group then generates a consolidated invoice.
+There are two types of billing groups:
+* Organization billing groups
+* Project billing groups
```

Comment:
I think the whole notion of "Project billing groups" should be removed, we don't really have those anymore, or it does not have a meaning once customer starts to use accounts.

Customer does not even see billing groups if they are "project billing groups".

I would just remove this whole section.